dtos/device.go

```diff
@@ -25,6 +25,7 @@ type Device struct {
 	ServiceName    string                        `json:"serviceName" validate:"required,edgex-dto-none-empty-string,edgex-dto-rfc3986-unreserved-chars"`
 	ProfileName    string                        `json:"profileName" validate:"required,edgex-dto-none-empty-string,edgex-dto-rfc3986-unreserved-chars"`
 	AutoEvents     []AutoEvent                   `json:"autoEvents,omitempty" validate:"dive"`
+	Protocol       string                        `json:"protocol,omitempty"`
```

Comment:
my bad, I forgot there is a `Protocols`.  They are too similar.
please rename this field to `ProtocolName`